content/techniques/rate-limiting.md

```diff
@@ -0,0 +1,80 @@
+### Rate Limiting
+
+In this chapter we cover how to add rate limiting to your application in a more "NestJS Way" than using a package like `express-rate-limit`. To get started, you'll need to install the `@nestjs/throttler` package.
+
+```bash
+$ npm i --save @nestjs/throttle
+```
+
+Once installation is complete, the `ThrottlerModule` can be configured as any other Nest package with `forRoot` or `forRootAsync`.
+
+```typescript
+@Module({
+  imports: [
+    ThrottlerModule.forRoot({
+      ttl: 60,
+      limit: 10,
+    }),
+  ]
+})
+export class AppModule {}
+```
+
+or
+
+```typescript
+@Module({
+  imports: [
+    ThrottlerModule.forRootAsync({
+      imports: [ConfigModule],
+      inject: [ConfigService],
+      useFactory: (config: ConfigService) => ({
+        ttl: config.get('THROTTLE_TTL'),
+        limit: config.get('THROTTLE_LIMIT'),
+      }),
+    }),
+  ],
+})
+export class AppModule {}
+```
+
+The above will set the global options for the `ttl`, the time to live, and the `limit`, the maximum number of requests within the ttl, for the routes of your application that are guarded.
+
+Once the module has been imported, you can then choose how you would like to bind the `ThrottlerGuard`. Any kind of binding as mentioned in the [guards](https://docs.nestjs.com/guards) section is fine. If you wanted to bind the guard globally, for example, you could do so but adding this provider to any module
```

Comment:
> If you wanted to bind the guard globally, for example, you could do so but adding this provider to any module

1) Add a dot to the end of the sentence.
2) Perhaps rephrase this sentence. What about this: _Binding the guard globally can be done by adding the `ThrottlerGuard` as global provider in the main app module._